package.json

```diff
@@ -23,36 +23,40 @@
     "node": ">=10.0.0"
   },
   "dependencies": {
-    "bcfg": "~0.1.6",
-    "bcrypto": "~3.1.11",
-    "bcurl": "^0.1.6",
-    "bdb": "~1.1.7",
-    "bdns": "~0.1.5",
-    "bevent": "~0.1.5",
-    "bfile": "~0.2.1",
-    "bfilter": "~1.0.5",
-    "bheep": "~0.1.5",
-    "binet": "~0.3.5",
-    "blgr": "~0.1.7",
-    "blru": "~0.1.6",
-    "blst": "~0.1.5",
-    "bmutex": "~0.1.6",
-    "bsert": "~0.0.10",
-    "bsip": "~0.1.9",
-    "bsock": "~0.1.9",
-    "bsocks": "~0.2.5",
-    "bstring": "~0.3.9",
-    "btcp": "~0.1.5",
-    "buffer-map": "~0.0.7",
-    "bufio": "~1.0.6",
-    "bupnp": "~0.2.6",
-    "bval": "~0.1.6",
-    "bweb": "~0.1.9",
-    "mrmr": "~0.1.8",
-    "n64": "~0.2.10"
+    "bcfg": "git+ssh://git@github.com:bcoin-org/bcfg.git#semver:~0.1.6",
+    "bcrypto": "git+ssh://git@github.com:bcoin-org/bcrypto.git#semver:~3.1.11",
+    "bcurl": "git+ssh://git@github.com:bcoin-org/bcurl.git#semver:^0.1.6",
+    "bdb": "git+ssh://git@github.com:bcoin-org/bdb.git#semver:~1.1.7",
+    "bdns": "git+ssh://git@github.com:bcoin-org/bdns.git#semver:~0.1.5",
+    "bevent": "git+ssh://git@github.com:bcoin-org/bevent.git#semver:~0.1.5",
+    "bfile": "git+ssh://git@github.com:bcoin-org/bfile.git#semver:~0.2.1",
+    "bfilter": "git+ssh://git@github.com:bcoin-org/bfilter.git#semver:~1.0.5",
+    "bheep": "git+ssh://git@github.com:bcoin-org/bheep.git#semver:~0.1.5",
+    "binet": "git+ssh://git@github.com:bcoin-org/binet.git#semver:~0.3.5",
+    "blgr": "git+ssh://git@github.com:bcoin-org/blgr.git#semver:~0.1.7",
+    "blru": "git+ssh://git@github.com:bcoin-org/blru.git#semver:~0.1.6",
+    "blst": "git+ssh://git@github.com:bcoin-org/blst.git#semver:~0.1.5",
+    "bmutex": "git+ssh://git@github.com:bcoin-org/bmutex.git#semver:~0.1.6",
+    "brq": "git+ssh://git@github.com:bcoin-org/brq.git#semver:~0.1.7",
+    "bs32": "git+ssh://git@github.com:bcoin-org/bs32.git#semver:=0.1.6",
+    "bsert": "git+ssh://git@github.com:chjj/bsert.git#semver:~0.0.10",
+    "bsip": "git+ssh://git@github.com:bcoin-org/bsip.git#semver:~0.1.9",
+    "bsock": "git+ssh://git@github.com:bcoin-org/bsock.git#semver:~0.1.9",
+    "bsocks": "git+ssh://git@github.com:bcoin-org/bsocks.git#semver:~0.2.5",
+    "bstring": "git+ssh://git@github.com:bcoin-org/bstring.git#semver:~0.3.9",
+    "btcp": "git+ssh://git@github.com:bcoin-org/btcp.git#semver:~0.1.5",
+    "buffer-map": "git+ssh://git@github.com:chjj/buffer-map.git#semver:~0.0.7",
+    "bufio": "git+ssh://git@github.com:bcoin-org/bufio.git#semver:~1.0.6",
+    "bupnp": "git+ssh://git@github.com:bcoin-org/bupnp.git#semver:~0.2.6",
+    "bval": "git+ssh://git@github.com:bcoin-org/bval.git#semver:~0.1.6",
+    "bweb": "git+ssh://git@github.com:bcoin-org/bweb.git#semver:=0.1.9",
```

Comment:
`bweb` latest is `0.1.10` when installed from master branch, is there a reason to keep this at `0.1.9` ?

Reply:
There are several that have patch updates. The reason is so that it matches the current lockfile.